src/app/components/media/MediaComponent.js

```diff
@@ -266,7 +266,6 @@ export default createFragmentContainer(withPusher(MediaComponent), graphql`
     project_id
     last_seen
     demand
-    articles_count
```

Comment:
Question: Are you sure you can delete this? It was being used for the Articles tab.

Reply:
Yes, it's needed. I just put it back.